plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/ObjectIdFunctions.java

```diff
@@ -52,6 +54,7 @@ public static Slice objectid(@SqlType(StandardTypes.VARCHAR) Slice value)
         return Slices.wrappedBuffer(new ObjectId(CharMatcher.is(' ').removeFrom(value.toStringUtf8())).toByteArray());
     }
 
+    @Description("Timestamp from the given Mongodb ObjectId")
     @ScalarFunction
     @SqlType("timestamp(3) with time zone") // ObjectId's timestamp is a point in time
```

Comment:
Should the return type be `timestamp(0) with time zone` here too?

Reply:
Makes sense, I've changed it.

Reply:
@rvazquezglez Please revert the change. Changing return type might be breaking change. We don't want to handle in this PR.

Reply:
Reverted.